easybuild/easyconfigs/s/SPH-EXA/SPH-EXA-v0.2.2-gompi-2021a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "CMakeMake"
+
+name = "SPH-EXA"
+version = "v0.2.2"
+
+homepage = "https://github.com/unibas-dmi-hpc/SPH-EXA_mini-app"
+description = """
+The SPH-EXA mini-app is a C++ scientific research code; its goal is to scale
+the Smoothed Particle Hydrodynamics (SPH) method to enable Tier-0 and Exascale
+simulations. More infos here: https://hpc.dmi.unibas.ch/en/research/sph-exa/
+"""
+
+toolchain = {"name": "gompi", "version": "2021a"}
+
+source_urls = [
+    "https://github.com/unibas-dmi-hpc/SPH-EXA_mini-app/archive/refs/tags"]
+sources = ["%(version)s.tar.gz"]
+checksums = [
+    "1e4c371d6783cc46698c4e3ed9ddc1c67236bb3e57da22cf16693954b69ac77f"]
+
+builddependencies = [
+    ("CMake", "3.20.1", "", ("GCCcore", "10.3.0")),
+]
+
+separate_build_dir = True
+
+install_cmd = """mkdir -p %(installdir)s/bin &&
+    cp %(builddir)s/SPH-EXA_mini-app*/LICENSE %(installdir)s/ &&
+    cp %(builddir)s/easybuild_obj/src/sedov/sedov %(installdir)s/bin/"""
+
+sanity_check_paths = {
+    "files": [
+        "bin/sedov",
```

Comment:
Please include `sedov --help` (or similar) as sanity check command

Reply:
Thanks for the comment. Currently, sedov can only run with mpi but I will add this --help flag to the next version. Or can we run the sanity command with mpirun ?